### ISSUE TITLE
10-7959a- react widget update

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#91185]
- update [react widgets](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/static-pages/ivc-champva) with approved/reviewed content from CAIA:
- CHAMPVA claims [CAIA ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82503) 
- Ensure that this form won't be indexed by search engines earlier than intended.
  - https://developers.google.com/search/docs/crawling-indexing/block-indexing
- Once this work is done, please comment on [this story](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106245) and tag @ManOnTheMooney that it can be closed

(Widget code is [located in this directory](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/static-pages/ivc-champva) of vets-website repo)

**Comment by andrea-merrill** [comment]
Waiting on content.  Requested in this [slack thread](https://dsva.slack.com/archives/C01K37HRUAH/p1742244669127069)

fysa: @michaelclement @ManOnTheMooney

**Comment by michaelclement** [comment]
Content provided [here](https://dsva.slack.com/archives/C01K37HRUAH/p1743531696184179?thread_ts=1742244669.127069&cid=C01K37HRUAH).

**Comment by michaelclement** [comment]
Work is done in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/35628). Waiting on:
- [ ] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
Work is done in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/35628). Waiting on:
- [ ] Team approval
- [ ] Platform approval

**Comment by michaelclement** [comment]
@ManOnTheMooney This has been merged in. To review, please see the screenshots above. Before CAIA can insert this into their Drupal page we'll want CHAMPVA Claims to be released to production ([separate issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106563)).

**Comment by ManOnTheMooney** [comment]
Looks great! Thanks Michael!
